src/Microsoft.AspNetCore.Http.Connections/ConnectionsRouteBuilder.cs

```diff
@@ -10,20 +10,39 @@
 
 namespace Microsoft.AspNetCore.Http.Connections
 {
+    /// <summary>
+    /// Maps routes to the HTTP connection dispatcher.
+    /// </summary>
     public class ConnectionsRouteBuilder
     {
         private readonly HttpConnectionDispatcher _dispatcher;
         private readonly RouteBuilder _routes;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ConnectionsRouteBuilder"/> class.
+        /// </summary>
+        /// <param name="routes">The underlying <see cref="RouteBuilder"/>.</param>
+        /// <param name="dispatcher">The dispatcher.</param>
         public ConnectionsRouteBuilder(RouteBuilder routes, HttpConnectionDispatcher dispatcher)
         {
             _routes = routes;
             _dispatcher = dispatcher;
         }
 
+        /// <summary>
+        /// Maps incoming requests with the specified path to the dispatcher.
+        /// </summary>
+        /// <param name="path">The request path.</param>
+        /// <param name="configure">A callback to configure the connection.</param>
         public void MapConnections(PathString path, Action<IConnectionBuilder> configure) =>
             MapConnections(path, new HttpConnectionDispatcherOptions(), configure);
 
+        /// <summary>
+        /// Maps incoming requests with the specified path to the dispatcher.
```

Comment:
"Maps incoming requests with the specified path to the provided connection pipeline"